images/initials.go

```diff
@@ -0,0 +1,84 @@
+package images
+
+import (
+	"bytes"
+	"image/color"
+	"image/png"
+	"io/ioutil"
+	"strings"
+
+	"github.com/fogleman/gg"
+	"golang.org/x/image/font"
+	"golang.org/x/image/font/opentype"
+)
+
+type RGBA struct {
+	R, G, B, A float64
+}
+
+var parsedFont *opentype.Font = nil
+
+func ExtractInitials(fullName string, amountInitials int) string {
+	if fullName == "" {
+		return ""
+	}
+	var initials strings.Builder
+	namesList := strings.Fields(fullName)
+	for _, name := range namesList {
+		if len(initials.String()) >= amountInitials {
+			break
+		}
+		if name != "" {
+			initials.WriteString(strings.ToUpper(name[0:1]))
+		}
+	}
+	return initials.String()
+}
+
+// uppercaseRatio is <height of any upper case> / dc.FontHeight() (line height)
+// 0.60386123 for Inter-UI-Medium.otf
+func GenerateInitialsImage(initials string, bgColor, fontColor color.Color, fontFile string, size int, fontSize float64, uppercaseRatio float64) ([]byte, error) {
```

Comment:
Just a little note here. The golang convention is to start all documentation with the name of the element. See example below:

```suggestion
// GenerateInitialsImage uppercaseRatio is <height of any upper case> / dc.FontHeight() (line height)
// 0.60386123 for Inter-UI-Medium.otf
func GenerateInitialsImage(initials string, bgColor, fontColor color.Color, fontFile string, size int, fontSize float64, uppercaseRatio float64) ([]byte, error) {
```